prow/external-plugins/refresh/main.go

```diff
@@ -54,8 +52,8 @@ func (o *options) Validate() error {
 		}
 	}
 
-	if _, err := url.ParseRequestURI(o.prowURL); err != nil {
-		return fmt.Errorf("invalid -prow-url URI: %q", o.prowURL)
+	if len(o.prowURL) > 0 {
+		logrus.Warn("--prow-url is deprecated, set prow_url in config.yaml instead")
```

Comment:
Do we need to go through the deprecation policy for this?

Reply:
I doubt anyone but me uses `refresh` but yes, we can default here.